requirements.txt

```diff
@@ -7,7 +7,7 @@ pandas==1.3.4
 psycopg2-binary==2.8.6
 pyppeteer==0.2.2
 pytest==6.1.2
-requests==2.24.0
+requests==2.27.1
```

Comment:
I had to update this to avoid a urllib3 dependency confilct between requests and prefect.